### PR TITLE
Add test for revcount, update docstring

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

File: base/libgit2/libgit2.jl [hunk 1]
```diff
@@ -484,7 +484,7 @@ function cat(repo::GitRepo, spec)
     end
 end
 
-""" git rev-list --count <commit1> <commit2> """
+""" git rev-list --left-right --count <commit1> <commit2> """
```
Comment (ararslan) [inline base/libgit2/libgit2.jl]: Any chance while you're in here you could make this an actual description of the functionality?

**Review by ararslan — APPROVED** [review]
💯 _Fantastic_ docstring.

File: base/libgit2/libgit2.jl [hunk 2]
```diff
@@ -484,7 +484,17 @@ function cat(repo::GitRepo, spec)
     end
 end
 
-""" git rev-list --count <commit1> <commit2> """
+"""
+    LibGit2.revcount(repo::GitRepo, fst::AbstractString, snd::AbstractString)
```
Comment (tkelman) [inline base/libgit2/libgit2.jl]: are fst and snd significant names? maybe something more anonymous would make sense for the docstring if there's no reference to anything about the arg names

Comment (ararslan) [inline base/libgit2/libgit2.jl]: The arg names are mentioned in the docstring, both in prose and in the command line Git equivalent at the bottom. Would `left` and `right` be accurate names?

Comment (StefanKarpinski) [inline base/libgit2/libgit2.jl]: I initially read these as "fast" and "send" which was very confusing.

**Comment by kshyatt** [comment]
Fixed conflict and changed confusing names.

**Comment by kshyatt** [comment]
Good to go?

**Pull request merged**